org.mixedrealitytoolkit.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyTouchAdapter.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Mixed Reality Toolkit Contributors
```

Comment:
Please add change comment to "UX Core's" CHANGELOG.md.   Following the [Keep A Changelog](https://keepachangelog.com/en/1.1.0/) pattern.

Also please change that "UX Core's" version has been properly incremented since last [release](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/releases).

Reply:
This should also be "version": "3.2.0-development". Thank you!

Reply:
This is already 3.2.0-development!